cinema_app/settings.py

```diff
@@ -0,0 +1,125 @@
+"""
+Django settings for cinema_app project.
+
+Generated by 'django-admin startproject' using Django 5.1.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-54&z481$(*--vs*(9ph!2_1xvj(_#w@0e(limw1d1=g@e8#(31"

```

Comment:
The `SECRET_KEY` should be kept secret in production. Consider using environment variables or a secrets manager to securely manage this key.